### DISCUSSION TITLE
question on updates

### DISCUSSION THREAD

**Discussion by legaltextai** [freelawproject/courtlistener#3562] [Q&A]
General question on the new bulk files: Is the only difference b/w July and December .csv opinion files the addition of the new opinions that were issued after the last decision in the previous (July) update, or it may include older opinions too? 
E.g. I am working with Indiana opinions. Last date for Indiana in July .csv update was 09/21. Is the new December .csv update is a sum of what was in July update + new opinions issued after 09/21, or it may include also opinions not included in the previous update but that were issued prior to 09/21? Thanks

**Comment by mlissner** [comment]
Every bulk data file is the complete data set, not a delta of the previous data.

**Reply by legaltextai** [reply]
I understand that. My question was about the actual difference between old bulk data set and the new one. Does the new one include any old cases that were not in the previous one?

**Reply by mlissner** [reply]
Every bulk data file has everything in the system at the time it is created, so any data we add since the last creation will be in the new file. 

So the related question is: "Do you add historical content?" and the answer to that is that we do add such content quite frequently.

**Reply by legaltextai** [reply]
Got it. Thanks.

**Reply by amitnagar** [reply]
Following up on the above question: 

Based on the response, I assume it is sufficient to pick the latest opinions archive (for example, opinions-2024-03-11.csv.bz2 as of today (4/15/2024)) to get access to all the opinions available in the system,

Thanks

**Reply by mlissner** [reply]
Yes. Files are snapshots, not deltas.

**Reply by amitnagar** [reply]
Thank You.
